frontend/src/screens/channels/Channels.tsx

```diff
@@ -69,6 +70,7 @@ export default function Channels() {
   const { data: channels, mutate: reloadChannels } = useChannels();
   const { data: nodeConnectionInfo } = useNodeConnectionInfo();
   const { data: balances, mutate: reloadBalances } = useBalances();
+  const { data: bitcoinRate } = useBitcoinRate("usd");
```

Comment:
Does it make sense to pass the currency from the frontend? should it be something a user can change in the settings page?

Reply:
hmm. nice idea. we can fetch supported currencies and show it in settings and use it to show balances + fait equivalent of transactions as well.  maybe we do it in separate pr?

Reply:
I think so, but in that case can you remove the query parameter passing that is in this PR?